bpdm-gate-api/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/api/GateAddressApi.kt

```diff
@@ -145,4 +145,20 @@ interface GateAddressApi {
     @PutExchange("/output/addresses")
     fun putAddressesOutput(@RequestBody addresses: Collection<AddressGateOutputRequest>): ResponseEntity<Unit>
 
+
+    @Operation(
+        summary = "Get address csv file",
+        description = "Get address csv file."
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Address CSV file was successfully provided"),
+            ApiResponse(responseCode = "404", description = "Address CSV file was not successful", content = [Content()])
+        ]
+    )
+
+    @GetMapping("/input/addresses/csv")
+    @GetExchange("/input/addresses/csv")
+    fun getAddressToCsv(): ResponseEntity<Any?>
```

Comment:
Currently we have a mix on return types between responseEntity and no responseEntity. We would like to move away from responseEntity type. Please replace this with String type as it should return csv directly